server/keycloak.js

```diff
@@ -0,0 +1,121 @@
+const querystring = require('querystring');
+const KeyCloakConnect = require('keycloak-connect');
+const axios = require('axios');
+
+const regionMap = {
+  region_fraser: 'Fraser',
+  region_interior: 'Interior',
+  region_northern: 'Northern',
+  region_vancouver_coastal: 'Vancouver Coastal',
+  region_vancouver_island: 'Vancouver Island',
+};
+
+const defaults = {
+  'ssl-required': 'external',
+  'confidential-port': 0,
+  'verify-token-audience': true,
+  'use-resource-role-mappings': true,
+  'policy-enforcer': {},
+};
+
+class Keycloak { // Wrapper class around keycloak-connect
```

Comment:
This is nice to have Keycloak manager like this. It helps us to Mock this class for testing.